Sources/TSCUtility/Triple.swift

```diff
@@ -195,7 +199,12 @@ extension Triple {
     
     /// The file extension for static libraries.
     public var staticLibraryExtension: String {
-        return ".a"
+        switch abi {
```

Comment:
This seems wrong.  It isn't the ABI that dictates the extensions, but rather the platform.  A combination of OS & ABI perhaps, but not just the ABI (e.g., `(.windows, .cygus)` would use `.a`, but `(.windows, .msvc)` would use `.lib`).